core/src/main/scala/kafka/server/ClientQuotaManager.scala

```diff
@@ -58,6 +59,8 @@ object ClientQuotaManagerConfig {
   val DefaultQuotaWindowSizeSeconds = 1
   // Purge sensors after 1 hour of inactivity
   val InactiveSensorExpirationTimeSeconds  = 3600
+  val QuotaRequestPercentDefault = Int.MaxValue.toDouble
+  val NanosToPercentagePerSecond = 100.0 / TimeUnit.SECONDS.toNanos(1)
```

Comment:
Both `Ns` and `Nanos` are already used in the code. I chose `Nanos` everywhere in the PR since it stands out better from `Ms`, especially since I was changing some measurements from millis to nanos. But I am ok with switching to `Ns` if that is preferable.